src/sonic-host-services/scripts/hostcfgd

```diff
@@ -469,6 +464,11 @@ class HostConfigDaemon:
         self.config_db.subscribe('FEATURE', lambda table, key, data: self.feature_state_handler(key, data))
         self.config_db.subscribe('KDUMP', lambda table, key, data: self.kdump_handler(key, data))
 
+        syslog.syslog(syslog.LOG_INFO,
+                      "Waiting for systemctl to finish initialization")
+        self.wait_till_system_init_done()
+        syslog.syslog(syslog.LOG_INFO, "starting update of feature states")
```

Comment:
Suggest changing this message to `systemctl has finished initialization -- proceeding ...`

Reply:
update the log message